drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/case_management_assessment_loader.rb

```diff
@@ -8,44 +8,35 @@ module HmisExternalApis::TcHmis::Importers::Loaders
   class CaseManagementAssessmentLoader < CustomAssessmentLoader
     ASSESSMENT_DATE_COL = 'Date Taken'.freeze
 
-    # Based on the the data Gig extracted from the form definition
+    # Actual values cleared, retaining the loader just for reference
```

Comment:
retained as example of custom assessment import

Reply:
Are any of the other of these worth retaining?  We can always recover them out of the repo, but maybe a couple examples would be helpful for future migrations.

Reply:
I included one per type, I think. One for custom assessments and one for services, and the "meal" one just because it was a little different.